### ISSUE TITLE
any possibility to develop a hash like redis hset/hget/hdel data type?

### ISSUE THREAD

**Issue by unisqu** [coocood/freecache#47]
i like this software and i know it's fast but i would like to have hash data structure like redis + ability to expire the hash key

is it in the production roadmap? it'll be a great add on feature.

**Comment by unisqu** [comment]
i really like this software more than the other memory caching, i can buy 2 cups of coffee for you on this as a token of appreciation. need hash like redis data structure for freecache and this software is perfect. hexpire on key too. thanks in advance.

**Comment by unisqu** [comment]
Can I pay you to develop this feature?

**Comment by coocood** [comment]
No, I'm sorry, I think this feature is not necessary and I don't have time to do it.
You can build a hash structure based on the basic key/value feature at the application level.

**Comment by unisqu** [comment]
how do i do hdel "key" for all hash structures? any ideas?
